development/build/mmi-scripts/build-for-all-custodians.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
We are slowly converting the codebase to TypeScript so if it is possible for this to be in TypeScript that would be appreciated! Note that you will want to run this script using `yarn ts-node`. You can save from having to remember this by adding a shebang at the top (something like `#!yarn ts-node`) and making this file executable (`chmod +x development/build/mmi-scripts/build-for-all-custodians.js`). Then you can  just run this file by saving `development/build/mmi-scripts/build-for-all-custodians.ts`.

Reply:
Thanks @mcmire, makes sense, let us update the file and I'll get back to you! 💪🏼